src/saveload/saveload.cpp

```diff
@@ -199,10 +199,10 @@ struct SaveLoadParams {
 	bool expect_table_header;            ///< In the case of a table, if the header is saved/loaded.
 
 	MemoryDumper *dumper;                ///< Memory dumper to write the savegame to.
-	SaveFilter *sf;                      ///< Filter to write the savegame to.
+	std::shared_ptr<SaveFilter> sf; ///< Filter to write the savegame to.
 
 	ReadBuffer *reader;                  ///< Savegame reading buffer.
-	LoadFilter *lf;                      ///< Filter to read the savegame from.
+	std::shared_ptr<LoadFilter> lf; ///< Filter to read the savegame from.
```

Comment:
Comment alignment?

Reply:
Keeping these trailing comments aligned over time is quite a hassle, so some of us have stopped bothering with it.
Essentially some of us have just been removing the spaces when we were manually changing the lines, without touching the other lines.